pages.ru/common/alias.md

```diff
@@ -0,0 +1,29 @@
+# alias
+
+> Создает псевдоним -- слова, которые будут заменены командой.
+> Срок действия псевдонима истекает с текущей сессией командной строки если только не определить его в конфигурационном файле, например: `~/.bashrc`.
+> Больше информации: <https://tldp.org/LDP/abs/html/aliases.html>.
+
+- Вывести список всех псевдонимов:
+
+`alias`
+
+- Создать типовой псевдоним:
+
+`alias {{псевдоним}}="{{команда}}"`
+
+- Вывести команду сопоставленную с данным псевдонимом:
+
+`alias {{псевдоним}}`
+
+- Удалить псевдоним:
+
+`unalias {{псевдоним}}`
+
+- Превратить `rm` в интерактивную команду:
+
+`alias {{rm}}="{{rm -i}}"`
```

Comment:
```suggestion
`alias {{rm}}="{{rm --interactive}}"`
```

Reply:
In this scenario, keep the same example as the main English page.